src/filestore/common.ts

```diff
@@ -28,18 +28,28 @@ export interface KinveyResponseConfig {
 export class CommonFileStore extends CoreFileStore {
   upload(file: File, metadata: any, options: any)
   upload(filePath: string, metadata: any, options: any)
-  upload(filePath: string | File, metadata: any, options: any) {
+  upload(filePath: string | File, metadata = <any>{}, options: any) {
```

Comment:
The syntax I like better for the default value is
`upload(filePath: string | File, metadata: any = {}, options: any)`

Also, isn't the metadata of type `FileMetadata`?